opentreemap/treemap/js/src/mapFeature.js

```diff
@@ -135,6 +136,7 @@ exports.init = function(options) {
     clickedIdStream
         .filter(BU.not, options.featureId)
         .map(_.partialRight(U.appendSegmentToUrl, detailUrlPrefix, false))
+        .filter(R.not(currentMover.isEnabled))
```

Comment:
You could consider defining `isDisabled()` instead of `isEnabled()` and removing this `R.not`